opal/threads/wait_sync.c

```diff
@@ -94,8 +94,10 @@ int ompi_sync_wait_mt(ompi_wait_sync_t *sync)
     /* In case I am the progress manager, pass the duties on */
     if( sync == wait_sync_list ) {
         wait_sync_list = (sync == sync->next) ? NULL : sync->next;
-        if( NULL != wait_sync_list )
+        if( NULL != wait_sync_list ){
+            /* This is a possible placement for an MPI_T progress switch counter */
```

Comment:
Do you think it is worthwhile to leave this comment in here?  If not, I can remove these changes.